examples/api-provider-sample/src/plugin/gotd-api-init.ts

```diff
@@ -31,7 +31,6 @@ import { ApiFactory, PluginContainerModule } from '@theia/plugin-ext/lib/plugin/
 
 type Gotd = typeof gotd;
 const GotdApiFactory = Symbol('GotdApiFactory');
-type GotdApiFactory = ApiFactory<Gotd>;
```

Comment:
Explanation: this type seems extraneous.

Reply:
So it is! There's no problem of unused name reported by tsc/eslint because the const symbol of the same name is referenced.